Makefile

```diff
@@ -96,6 +97,8 @@ build-all: build-cli-${GOHOSTOS}-${GOHOSTARCH} build-publish-admin-plugins-local
 
 .PHONY: build
 build: build-cli-${GOHOSTOS}-${GOHOSTARCH} ## Build the Tanzu Core CLI for the local platform
+	mkdir -p bin
+	cp $(ARTIFACTS_DIR)/$(GOHOSTOS)/$(GOHOSTARCH)/cli/core/$(BUILD_VERSION)/tanzu-cli-$(GOHOSTOS)_$(GOHOSTARCH) ./bin/tanzu
```

Comment:
`build-all` doesn't call `build` at the moment, which means it would get the `bin/tanzu` binary.
Can you change the dependencies for `build-all` to depend on `build` instead of `build-cli-${GOHOSTOS}-${GOHOSTARCH}`

Reply:
updated